tools/build-release-branch.sh

```diff
@@ -52,6 +52,12 @@ function create_release_gitignore {
 
     # Add custom stuff to .gitignore release
     echo "/_inc/client" >> .gitignore
+    # Needs to stay in sync with .svnignore
+    # The quoting is important here so the shell expands the contents producing multiple strings.
+    # .gitignore will not respect the {brace,variations} so we'll produce multiple lines.
+    #
+    #                       ↓ this brace is unquoted. Important!
+    printf "%s\n" "/extensions/**/*."{"js","jsx","json","gif","png","css","jpg","jpeg","sass","scss","svg"} >> .gitignore
```

Comment:
Alternatively, we could simplify this to just echoing multiple lines or looping a simple array. That might help to catch any issues and help people to expand this if need.

Reply:
Good call. Simplified in e450f018a3612268e293d299a4f144948ddf3bc5